flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/stream/table/TableAggregateTest.scala

```diff
@@ -0,0 +1,159 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.table.api.stream.table
+
+import org.apache.flink.api.scala._
+import org.apache.flink.table.api.scala._
+import org.apache.flink.table.expressions.utils.Func0
+import org.apache.flink.table.utils.{EmptyTableAggFunc, TableTestBase, TopN}
+import org.apache.flink.table.utils.TableTestUtil._
+import org.junit.Test
+
+class TableAggregateTest extends TableTestBase {
+
```

Comment:
I have not supported alias since this will change `def flatAggregate(tableAggCall: TableAggFunctionCall)` to `def flatAggregate(tableAggCall: Expression)`. Maybe we can make `as` as a method for TableAggFunctionCall. This not only force a TableAggFunctionCall but also support alias?